lang/de/validation.php

```diff
@@ -0,0 +1,721 @@
+        'numbers' => ':Attribute muss mindestens eine Zahl beinhalten.',
+        'symbols' => ':Attribute muss mindestens ein Sonderzeichen beinhalten.',
+        'uncompromised' => ':Attribute wurde in einem Datenleck gefunden. Bitte wählen Sie ein anderes :attribute.',
+    ],
+    'present' => ':Attribute muss vorhanden sein.',
+    'present_if' => 'Das Feld :attribute muss vorhanden sein, wenn :other gleich :value ist.',
+    'present_unless' => 'Das Feld :attribute muss vorhanden sein, es sei denn, :other ist :value.',
+    'present_with' => 'Das Feld :attribute muss vorhanden sein, wenn :values vorhanden ist.',
+    'present_with_all' => 'Das Feld :attribute muss vorhanden sein, wenn :values vorhanden sind.',
+    'prohibited' => ':Attribute ist unzulässig.',
+    'prohibited_if' => ':Attribute ist unzulässig, wenn :other :value ist.',
+    'prohibited_unless' => ':Attribute ist unzulässig, wenn :other nicht :values ist.',
+    'prohibits' => ':Attribute verbietet die Angabe von :other.',
+    'regex' => ':Attribute Format ist ungültig.',
+    'required' => ':Attribute muss ausgefüllt werden.',
+    'required_array_keys' => 'Dieses Feld muss Einträge enthalten für: :values.',
+    'required_if' => ':Attribute muss ausgefüllt werden, wenn :other den Wert :value hat.',
+    'required_if_accepted' => ':Attribute muss ausgefüllt werden, wenn :other gewählt ist.',
+    'required_if_declined' => 'Das Feld :attribute ist erforderlich, wenn :other abgelehnt wird.',
+    'required_unless' => ':Attribute muss ausgefüllt werden, wenn :other nicht den Wert :values hat.',
+    'required_with' => ':Attribute muss ausgefüllt werden, wenn :values ausgefüllt wurde.',
+    'required_with_all' => ':Attribute muss ausgefüllt werden, wenn :values ausgefüllt wurde.',
+    'required_without' => ':Attribute muss ausgefüllt werden, wenn :values nicht ausgefüllt wurde.',
+    'required_without_all' => ':Attribute muss ausgefüllt werden, wenn keines der Felder :values ausgefüllt wurde.',
+    'same' => ':Attribute und :other müssen übereinstimmen.',
+    'size' => [
+        'array' => ':Attribute muss genau :size Elemente haben.',
+        'file' => ':Attribute muss :size Kilobyte groß sein.',
+        'numeric' => ':Attribute muss gleich :size sein.',
+        'string' => ':Attribute muss :size Zeichen lang sein.',
+    ],
+    'starts_with' => ':Attribute muss mit einem der folgenden Anfänge aufweisen: :values',
+    'string' => ':Attribute muss ein String sein.',
+    'timezone' => ':Attribute muss eine gültige Zeitzone sein.',
+    'ulid' => 'Die :attribute muss eine gültige ULID sein.',
+    'unique' => ':Attribute ist bereits vergeben.',
+    'uploaded' => ':Attribute konnte nicht hochgeladen werden.',
+    'uppercase' => ':Attribute muss in Großbuchstaben sein.',
+    'url' => ':Attribute muss eine URL sein.',
+    'uuid' => ':Attribute muss ein UUID sein.',
+
+    'attributes' => [
+        'Typ' => 'Typ',
```

Comment:
adjust this according to the changes in lang/en/validation.php (If this gets changes/removed).